safe/impact_function/provenance_utilities.py

```diff
@@ -28,17 +28,17 @@ def get_map_title(hazard, exposure, hazard_category):
     :rtype: str
     """
     if hazard == hazard_generic:
-        map_title = tr('{exposure_name} affected'.format(
+        map_title = tr(u'{exposure_name} affected'.format(
```

Comment:
Perhaps I was wrong. But should we do tr first before doing the formating? I know I am the one who write the code, but I just realised that probably I was wrong.